src/posts/pitfalls-of-accessible-components.md

```diff
@@ -0,0 +1,90 @@
+Content also goes beyond paragraphs and text blocks. Don't forget your alternate texts for images, or captions and transcripts for other media files.
+
+### The page title
+
+Match this as much as possible with the main heading on your page (the `<h1>`) and the names of the links pointing to the page. If there are repeating parts in the title (such as the name of the website), place them as far back in the title as possible. This makes scanning (also with screen readers) easier. The unique and distinctive part comes first.
+In a webshop you get something like: “Product - category - webShopName.nl”
+
+## Page-specific: structure
+
+Perhaps you have something reusable for this, such as templates. That's very nice. However, experience shows that these are not always available and do not have priority.
+
+### Landmarks
+
+You want to have your landmarks in order for each page:
+- Place your navigation in a `<nav>`
+- Place the most important content in a `<main>`
+- Use a `<header>` at the beginning of your page and a `<footer>` at the end
+- Do you have information aside? Then use a `<aside>`
+
+### Heading structure
+
+Your headings help people navigate through your page. Just like you can scan headlines in a newspaper, you want to do the same on a website.
+The `<h1>` is for the main heading; for what your page is about. Below that you want to create a kind of tree structure of your headings. All content logically (both visually and in code) falls under one of your headings. You go from main topic (`<h1>`) to sub-topic (`<h2>`), to sub-sub-topic (`<h3>`), and so on. Never move forward more than one step at a time.
+
+### Focus Order
+
+Make sure that every page you tab through navigates logically. This is easier if you don't use a tab index with a value higher than 0. Creative application of CSS grid or flexbox can also cause the visual order to differ from that in the code.
+Do you want to make it completely easy for your visitor? Add a skip link. That is a (often hidden) link at the beginning of the page that allows you to "jump" to the content.
+
+## Difficult to test
+
+### Variations of components
+
+The more features a component has, the greater the chance that it will go wrong. If a component has a boolean attribute, dark mode and two responsive zoom levels, then you quickly end up with (2x2x3=8) eight variations that all need to be tested and maintained. So pay attention to this! If everyone around you has default dark mode on, you can count on issues in light mode.
+
+### Composition
+
+As soon as you start combining components, you can also foresee problems. You would prefer, for example, that forms have been developed and bundled as a complete system. You often combine the same components in forms. If they are delivered separately, unexpected (and untested) situations may arise. Another point of attention!
```

Comment:
>As soon as you start combining components

This is the first time we've really mentioned components. I'm thinking the intro might need a bit more detail to clarify what I'm guessing is the structure of the post.